sdk/textanalytics/Azure.AI.TextAnalytics/src/OperationHelpers.cs

```diff
@@ -0,0 +1,62 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Threading;
+using System.Threading.Tasks;
+
+#nullable enable
+
+namespace Azure.Core
```

Comment:
why do we have this class here? If it belongs in`Azure.Core1 we shouldn't create a copy.
If the one in Core gets updated, we will miss the fix or will have to manually update it.

Reply:
the prefer way to do it is to add it as a shared source in [here](https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/textanalytics/Azure.AI.TextAnalytics/src/Azure.AI.TextAnalytics.csproj#L17)